src/csv/CHERI_ISA.csv

```diff
@@ -41,7 +41,7 @@
 "CGETHIGH","✔","✔","","","","","✔","Both","","","","","","","","","","","","","","","","OP","1-src 1-dst","","","Get metadata","","","","","","","",""
 "CSETHIGH","✔","✔","","","","","✔","Both","","","","","","","","","","","","","","","","OP","R-type","","","Set metadata and clear tag","","","","","","","",""
 "CSETEQUALEXACT","✔","✔","","","","","✔","Both","","","","","","","","","","","","","","","","OP","R-type","","","Full capability bitwise compare","","","","","","","",""
-"CSEAL","✔","✔","","","","","✔","Both","","","","","","","","","","","","","","","","OP","R-type","","","Seal capability","","","","","","","",""
+"CSEALENTRY","✔","✔","","","","","✔","Both","","","","","","","","","","","","","","","","OP","R-type","","","Seal capability","","","","","","","",""
```

Comment:
These are files are downloaded from a spreadsheet in Google Drive. Is the original spreadsheet up to date?